### PR TITLE
Upload byte Img to Url

### LINKED ISSUE #67: upload image

### PR DESCRIPTION
**Pull request by wj461**
when post product
need send byte img for CartIngo.picture

### CODE REVIEW

**Comment by wj461** [comment]
CartIngo -> CartInfo

**Review by nonorze-VOXX — CHANGES_REQUESTED** [review]
INWOMM

File: backend/product/usecase/product_usecase.go [hunk 1]
```diff
@@ -44,3 +49,35 @@ func (pu *productUsecase) UpdateById(ctx context.Context, StoreId int64, product
...
+	s := domain.PostInfo{
+		Key:    "6d207e02198a847aa98d0a2a901485a5",
+		Action: "upload",
+		Source: byteImg,
+		Format: "json"}
+	v := make(map[string][]string)
+	v["key"] = []string{s.Key}
+	v["action"] = []string{s.Action}
+	v["source"] = []string{s.Source}
+	v["format"] = []string{s.Format}
```
Comment (NOOBDY) [inline backend/product/usecase/product_usecase.go:65]:
```suggestion
	v := map[string][]string{
	  "key": []string{s.Key},
	  "action": []string{s.Action},
	  "source": []string{s.Source},
	  "format": []string{s.Format},
	}
```

File: backend/domain/product.go [hunk 2]
```diff
@@ -6,6 +6,23 @@ import (
...
+
+type ResponseInfo struct {
+	StatusCode int       `json:"status_code"`
+	Image      ImageInfo `json:"image"`
+}
+
+type ImageInfo struct {
+	Url       string `json:"url"`
+	UrlViewer string `json:"url_viewer"`
+}
```
Comment (NOOBDY) [inline backend/domain/product.go:24]:
```suggestion
type ResponseInfo struct {
	StatusCode int       `json:"status_code"`
	Image      struct {
	  Url       string `json:"url"`
	  UrlViewer string `json:"url_viewer"`
	} `json:"image"`
}
```

**Pull request closed without merging**